### PR TITLE
Rules

### PR DESCRIPTION
**Pull request by codewithrishu**
made rules page

deploy link - https://zealous-mahavira-2594ed.netlify.app/rules/rules.html

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
the design and color contrast is good for rules page , good to merge !

File: rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.rule-points {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem;
+    background-color: var(--third-white);    
+    border-radius: 2rem;
+    margin: 1rem;
+
+
```
Comment (romabulani) [inline rules/rules.css:33]: If you have time, remove these additional lines. These are just minute details, nothing to worry about.

**Review by romabulani — APPROVED** [review]
Looks good to merge

File: rules/rules.css (hunk 1 above)
Comment (codewithrishu) [reply rules/rules.css:33]: done :)

**Pull request merged**